Include/internal/_mem.h

```diff
@@ -11,6 +11,9 @@ extern "C" {
 #include "_pymalloc.h"
 #endif
 
+#undef  uint
+#define uint    unsigned int    /* assuming >= 16 bits */
```

Comment:
It's the original code that I forgot to copy over. :)  I don't mind fixing it up though.